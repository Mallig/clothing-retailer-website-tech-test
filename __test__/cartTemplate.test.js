const pug = require('pug')
const cartTotal = { total: 100 }

const compiledFunction = pug.compileFile('views/cart.pug')

test('It should render pug correctly', () => {
  expect(compiledFunction(cartTotal)).toMatchSnapshot();
});