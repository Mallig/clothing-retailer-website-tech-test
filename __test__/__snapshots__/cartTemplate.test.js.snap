// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render pug correctly 1`] = `"<d> <p>Welcome to the Basket</p><a href=\\"/\\">Shop</a></d><p>Cart Total: Â£100</p><form method=\\"POST\\" action=\\"cart/discount\\"><input name=\\"voucher_code\\" type=\\"text\\"/><button>Apply Voucher</button></form>"`;
